scipy/stats/_distribution_infrastructure.py

```diff
@@ -3497,17 +3517,21 @@ def make_distribution(dist):
 
             endpoints : tuple
                 A tuple defining the lower and upper endpoints of the domain of the
-                parameter. Allowable values are floats or the name (string) of another
+                parameter; allowable values are floats or the name (string) of another
```

Comment:
Oops. I didn't mean to change this. This happened because I'd accidentally documented `endpoints` as taking a method here instead of `support`, and the semicolon seemed to make things flow better after adding an additional sentence on `method` values of `endpoints`, since it make the linking of the first two sentence more clear.

```suggestion
                parameter. Allowable values are floats or the name (string) of another
```